src/models/Response.ts

```diff
@@ -18,6 +18,12 @@ export interface Product {
   remainderInterestPeriodMontlyRepayment: number
 }
 
+export interface Terms {
+  standard?: string
+  hybrid?: string
+  fixed?: string
+}
```

Comment:
You may as well just reuse `TermProps`, right? Alternatively, `Terms` should expand `Termprops`